### PR TITLE
Circuitscape4

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@hajgato: this needs a sync with develop to fix a merge conflict

File: easybuild/easyconfigs/p/Python/Python-2.7.8-intel-2014b.eb [hunk 1]
```diff
@@ -90,6 +90,9 @@ exts_list = [
     ('matplotlib', '1.3.1', {
         'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib/'],
     }),
+    ('pyamg', '2.1.0', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.8-intel-2014b.eb]: maybe just create a dedicated easyconfig for this?

**Comment by hajgato** [comment]
@boegel Python module reverted, extra PyAMG module created.

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/389b21fe0952f4f333a2 for a full test report.

File: easybuild/easyconfigs/p/PyAMG/PyAMG-2.2.1-intel-2014b-Python-2.7.8.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+name = 'PyAMG'
+version = '2.2.1'
+
+homepage = 'http://pyamg.org'
+description = """PyAMG is a library of Algebraic Multigrid (AMG) solvers with a convenient Python interface."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyAMG/PyAMG-2.2.1-intel-2014b-Python-2.7.8.eb]: should be `SOURCELOWER` in both?

**Comment by hajgato** [comment]
@boegel Sorry for that, sourceLOWER added

File: easybuild/easyconfigs/p/PyAMG/PyAMG-2.2.1-intel-2014b-Python-2.7.8.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+name = 'PyAMG'
+version = '2.2.1'
+
+homepage = 'http://pyamg.org'
+description = """PyAMG is a library of Algebraic Multigrid (AMG) solvers with a convenient Python interface."""
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+
+source_urls = [PYPI_SOURCELOWER]
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyAMG/PyAMG-2.2.1-intel-2014b-Python-2.7.8.eb]: this should be `PYPI_LOWER_SOURCE`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/da149e5900740d3a234c for a full test report.

**Comment by boegel** [comment]
Looking great now, thanks @hajgato!

**Pull request merged**